### PR TITLE
Updated Oranges sell location

### PR DESCRIPTION
**Pull request by Khajit71**
changed paleto one

### CODE REVIEW

File: resources/[orange-picking]/Burevestnik_orangejob_lib/config.lua [hunk 1]
```diff
@@ -29,9 +29,9 @@ Config.Translation = {
 }
 
 Config.Blips = {
-    [1] = {["x"] = 416.5863, ["y"] = 6520.8721, ["z"] = 26.7128, ["id"] = 569, ["color"] = 52, ["scale"] = 0.7, ["text"] = "Orange Factory"},
-    [2] = {["x"] = 2564.1194, ["y"] = 4680.6182, ["z"] = 33.0768, ["id"] = 569, ["color"] = 52, ["scale"] = 0.7, ["text"] = "Orange Factory"},
-    [3] = {["x"] = -1933.6201, ["y"] = 2039.0726, ["z"] = 140.8295, ["id"] = 569, ["color"] = 52, ["scale"] = 0.7, ["text"] = "Orange Factory"},
+    [1] = {["x"] = 408.3538, ["y"] = 6497.1899, ["z"] = 27.8204, ["id"] = 569, ["color"] = 52, ["scale"] = 0.7, ["text"] = "Orange Factory"}, -- Paleto
```
Comment (minipunch) [inline resources/[orange-picking]/Burevestnik_orangejob_lib/config.lua:32]: Did u test to make sure this was good..............................

Comment (Khajit71) [reply resources/[orange-picking]/Burevestnik_orangejob_lib/config.lua:32]: nah, but what could go wrong LOL. It's only a coordinate change....

Comment (DatBoiWeep) [reply resources/[orange-picking]/Burevestnik_orangejob_lib/config.lua:32]: A lot can actually. LOL

Comment (minipunch) [reply resources/[orange-picking]/Burevestnik_orangejob_lib/config.lua:32]: It can be too high (z coord), even just a typo that a human might miss, etc

Comment (minipunch) [reply resources/[orange-picking]/Burevestnik_orangejob_lib/config.lua:32]: The point of testing is to cover the cases you can't think of lol

**Review by minipunch — COMMENTED** [review]
We should just use the `afarm` staff farm menu to reposition the farm that way

Don't need to change code for that

**Pull request closed without merging**